system-tests/__snapshots__/plugins_spec.js

```diff
@@ -421,7 +362,7 @@ Error: Async error from plugins file
 exports['e2e plugins fails when invalid event is registered 1'] = `
 Your configFile threw a validation error from: /foo/bar/.projects/plugin-validation-error/cypress.config.js
 
-You must pass a valid event name when registering a plugin.
+You must pass a valid event name when registering either a node event or plugin.
```

Comment:
May need to follow up with DX but I **think** we are still referring to things like

```js
on('someTask', { 
  // ...
})
```

As "plugins". We just don't have a `pluginsFile` anymore, but you can still write a plugin for Cypress.

Reply:
[`f453567` (#20560)](https://github.com/cypress-io/cypress/pull/20560/commits/f4535678adeca9ec037f379a1f016c9238b0b395)